src/app/components/header/header.component.ts

```diff
@@ -10,7 +10,7 @@ export class HeaderComponent implements OnInit {
   @Input() menu: Object[];
   @Input() title: string;
   @Output() toggleSidebar = new EventEmitter();
-  sidebarOpened: boolean = false;
+  sidebarOpened = false;
```

Comment:
This typing was throwing a lint error saying we should rely on implicit typing for booleans. We could bring it back but we'd need to fix our lint setup as well.